dataflow/src/test/java/livevar/LiveVariable.java

```diff
@@ -20,24 +16,9 @@
      * @param args not used
      */
     public static void main(String[] args) {
-
-        String inputFile = "Test.java";
```

Comment:
Would it be nicer to keep these variables here, like in the LVPlayground, and add these four parameters to `writeStringOfCFG`.
Then one can locally change the names here, where it is easier to find and the common code in `CFGVisualizeLauncher` can remain unchanged.

Reply:
Thanks, done.